### PR TITLE
bulk actions for repos

### LINKED ISSUE #55: Bulk actions for repos
In repo listing, admin can view repos of a single user and then select &ark them as approved/suspicious in one go.

### PR DESCRIPTION
**Pull request by avinash-mane**

### CODE REVIEW

File: src/pages/api/update-manual-review.js [hunk 1]
```diff
@@ -49,38 +49,41 @@ const clearRemark = async (id) => {
 
 //function for manual review
 const updateManualRepo = async (req, res) => {
-  const repoId = req.query.id;
+  let repoIds = req.query.ids.split(",");
```
Comment (sahilbhatia) [inline src/pages/api/update-manual-review.js]: Please explain why "let" was chosen here.

File: src/pages/api/update-manual-review.js [hunk 2]
```diff
@@ -49,38 +49,41 @@ const clearRemark = async (id) => {
...
+  repoIds = repoIds.map(Number);
```
Comment (sahilbhatia) [inline src/pages/api/update-manual-review.js]: Why do we need this? Can this not be avoided?

File: src/pages/api/update-manual-review.js [hunk 3]
```diff
@@ -49,38 +49,41 @@ const clearRemark = async (id) => {
...
-          });
-        } else {
-          const updatedRepo = await updateRepo(repoId, updatedAt);
-          if (updatedRepo[1].dataValues.parent_repo_id) {
-            await updateParentRepo(
-              updatedRepo[1].dataValues.parent_repo_id,
-              updatedAt
-            );
-            await clearRemark(updatedRepo[1].dataValues.parent_repo_id);
+        await repoIds.map(async (id) => {
```
Comment (sahilbhatia) [inline src/pages/api/update-manual-review.js]: Why do we need await before repoIds.map?

File: src/pages/api/update-manual-review.js [hunk 4]
```diff
@@ -49,38 +49,41 @@ const clearRemark = async (id) => {
...
+          let repo = await Repositories.findOne({ where: { id: id } });
+          if (!repo) {
+            res.status(404).json({
+              message: "Repository Not Found For Given Id",
```
Comment (sahilbhatia) [inline src/pages/api/update-manual-review.js]: Move all messages to locale file

File: src/pages/api/update-manual-review.js [hunk 5]
```diff
@@ -49,38 +49,41 @@ const clearRemark = async (id) => {
...
-          res.status(200).json({
-            message: "Repository Updated Successfully",
-          });
-        }
+        });
+        res.status(200).json({
+          message: "Repository Updated Successfully",
+        });
       } catch (err) {
         Sentry.captureException(err);
```
Comment (sahilbhatia) [inline src/pages/api/update-manual-review.js:93]: Configure once globally

**Review by sahilbhatia — CHANGES_REQUESTED** [review]
Note: Haven't explicitly commented on each occurance.

File: src/pages/api/update-manual-review.js (hunk 1 above)
Comment (avinash-mane) [reply src/pages/api/update-manual-review.js]: need to modify value

File: src/pages/api/update-manual-review.js (hunk 2 above)
Comment (avinash-mane) [reply src/pages/api/update-manual-review.js]: repo ids comes as a string array from request, we need to convert into in integer array

File: src/pages/api/update-manual-review.js (hunk 3 above)
Comment (avinash-mane) [reply src/pages/api/update-manual-review.js]: removed

File: src/pages/api/update-manual-review.js (hunk 4 above)
Comment (avinash-mane) [reply src/pages/api/update-manual-review.js]: constant file created

**Pull request closed without merging**